src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -365,6 +378,10 @@ async Task ExecuteInvocation()
                     catch (KeyNotFoundException) { }
                 }
             }
+
+            hubActivator?.Release(hub);
```

Comment:
Why did you move `hubActivator?.Release(hub);` to the end? Is it important?

Reply:
No, I just wanted the Release and Dispose to be together.